src/AutoRest.CSharp.V3/Properties/launchSettings.json

```diff
@@ -115,6 +115,10 @@
     "xml-service": {
       "commandLineArgs": "--standalone --input-codemodel=$(SolutionDir)\\test\\TestServerProjects\\xml-service\\CodeModel.yaml --plugin=csharpgen --output-folder=$(SolutionDir)\\test\\TestServerProjects\\xml-service --namespace=xml_service",
       "commandName": "Project"
+    },
+    "model-flattening": {
```

Comment:
use the generate.ps1 to update this, names should be in alphabetical order.

Reply:
Done.